{"name": "test_subFloat[0.4-0.3-0.1]", "status": "failed", "statusDetails": {"message": "AssertionError: 实际结果0.10000000000000003与预期结果0.1不一致\nassert 0.10000000000000003 == 0.1", "trace": "self = <pytestSecond.test.testCalculator.TestCalculator object at 0x03BB0D48>, a = 0.4, b = 0.3, expected = 0.1\n\n    @pytest.mark.run(order=2)\n    @pytest.mark.parametrize(['a', 'b', 'expected'], getData(\"sub\", \"floatDatas\"))\n    def test_subFloat(self, a, b, expected):\n        result = self.cal.sub(a, b)\n>       assert result == expected, f'实际结果{result}与预期结果{expected}不一致'\nE       AssertionError: 实际结果0.10000000000000003与预期结果0.1不一致\nE       assert 0.10000000000000003 == 0.1\n\ntestCalculator.py:109: AssertionError"}, "parameters": [{"name": "a", "value": "0.4"}, {"name": "b", "value": "0.3"}, {"name": "expected", "value": "0.1"}], "start": 1606204073498, "stop": 1606204073498, "uuid": "2f260231-39be-4566-af10-f9c60fdb14aa", "historyId": "eddba2d24f59beca81786bec853a3888", "testCaseId": "e248cf0aa6b3ea5618ef18dd781952b5", "fullName": "testCalculator.TestCalculator#test_subFloat", "labels": [{"name": "tag", "value": "run(order=2)"}, {"name": "suite", "value": "testCalculator"}, {"name": "subSuite", "value": "TestCalculator"}, {"name": "host", "value": "DESKTOP-103B3LA"}, {"name": "thread", "value": "47728-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCalculator"}]}